pkg/components/pluggable/grpc.go

```diff
@@ -103,6 +104,11 @@ func socketDialer(socket string, additionalOpts ...grpc.DialOption) GRPCConnecti
 // SocketDial creates a grpc connection using the given socket.
 func SocketDial(ctx context.Context, socket string, additionalOpts ...grpc.DialOption) (*grpc.ClientConn, error) {
 	udsSocket := "unix://" + socket
+
+	if runtime.GOOS == "windows" {
+		udsSocket = "unix:" + socket
```

Comment:
Could we add a test to cover it?

Reply:
I'll see what can be added.

Reply:
Found some more related tests to unblock on Windows.